code/controllers/subsystem/shuttle.dm

```diff
@@ -305,6 +305,9 @@ SUBSYSTEM_DEF(shuttle)
 	return 1
 
 /datum/controller/subsystem/shuttle/proc/autoEvac()
+	if (SSticker.current_state == GAME_STATE_FINISHED)
```

Comment:
if(!SSticker.IsRoundInProgress())

Reply:
I was thinking about letting it still happen in the preround or setup in case of weird corner situations, but I suppose if an admin caused it an admin can fix it...